content/tracing/setup/go.md

```diff
@@ -5,56 +5,86 @@ aliases:
 - /tracing/go/
 - /tracing/languages/go
 further_reading:
-- link: "https://github.com/DataDog/dd-trace-go"
+- link: "https://github.com/DataDog/dd-trace-go/tree/v1"
   tag: "Github"
   text: Source code
-- link: "https://godoc.org/github.com/DataDog/dd-trace-go/tracer"
+- link: "https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
   tag: "GoDoc"
   text: Package page
 - link: "tracing/visualization/"
   tag: "Documentation"
   text: "Explore your services, resources and traces"
 ---
 
-## Installation
+## Getting started
 
-To begin tracing applications written in Go, first [install and configure the Datadog Agent][1] (see additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+For configuration instructions and details about using the API, check out our [API documentation][api docs] for manual
+instrumentation, and our [integrations section][contrib docs] for libraries and frameworks supporting automatic instrumentation.
 
-Next, install the Go Tracer from the Github repository:
+For descriptions of terminology used in APM, take a look at the [Getting started with APM][getting started] section.
+
+For details about contributing, check the official repository [README.md file][repo readme].
+
```

Comment:
We should link our migration guide from here such as: "If you're using the Tracer 0.6.x, you can migrate to 1.0 following this migration guide" (or similar)

Reply:
done!